### ISSUE TITLE
TypeError: Expo is not a function

### ISSUE THREAD

**Issue by jlubeck** [expo/expo#2043]
I just installed expo-server-sdk with 

`yarn add expo-server-sdk`

If I try running my node app with this line:
`import Expo from 'expo-server-sdk';`

I'm getting this error

```
index.js:15
import Expo from 'expo-server-sdk';
       ^^^^

SyntaxError: Unexpected identifier
    at new Script (vm.js:74:7)
    at createScript (vm.js:246:10)
    at Object.runInThisContext (vm.js:298:10)
    at Module._compile (internal/modules/cjs/loader.js:670:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)
    at startup (internal/bootstrap/node.js:238:19)
```

If I try the old require way

`const Expo = require('expo-server-sdk');`

I'm getting this other error

```
index.js:16
let expo = new Expo();
           ^

TypeError: Expo is not a constructor
    at Object.<anonymous> (/Users/jan/DropboxPersonal/Dropbox/Projects/Expo/my-home-server/index.js:16:12)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)
    at startup (internal/bootstrap/node.js:238:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:572:3)
```

Am I doing something wrong?

Thank you!

**Comment by jlubeck** [comment]
Well... that was stupid...

`const { Expo } = require('expo-server-sdk');`

That worked

**Comment by muniperez** [comment]
Hey, guys, this is very confusing. I'm facing the same problem.

I'm following the exact instructions from the Readme section but when  I updated from version 2 to version 3, I started getting this error - but the documentation didn't change.

With version ^2.3.3 I do the following:

`
const Expo = require('expo-server-sdk')
let expo = new Expo();

// Check if tokens are valid for each device
if(Expo.isExpoPushToken(token)) {
 // Add this devace to a list of notifications
}
... // send the notifications
`
Apparently, the Expo constructor is not a constructor anymore (???)

When I do the same thing with version ^3, I get:

`
TypeError: Expo is not a constructor
Expo.isExpoPushToken is not a function
`

How should we use the API now?
